packages/server/app/controllers/config_controller.ts

```diff
@@ -105,6 +109,47 @@ export default class ConfigController {
     return { data: featureMenuConfigWithTypes }
   }
 
+  /**
+   * @filterConfig
+   * @summary Filter configuration
+   * @description Returns a payload with configuration information for filterable fields for the
+   * various data types.
+   */
+  async filterConfiguration({}: HttpContext) {
+    const result: any = {}
+    const models = [
```

Comment:
Let's move these to a filter.ts or search.ts(?) in the package/serverr/config directory.  change the name to filterableModels or something similar.  Seems like it might be useful to have it in one place.